recipes/tinyobjloader/1.x/conanfile.py

```diff
@@ -0,0 +1,51 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class TinyObjLoaderConan(ConanFile):
+    name = "tinyobjloader"
+    description = "Tiny but powerful single file wavefront obj loader - Builds on Windows, Linux and Macos/OSX"
+    settings = "os", "arch", "build_type", "compiler"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {'shared': False, "fPIC": True}
+    license = "MTI"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/syoyo/tinyobjloader"
+    author = "Florian Stellbrink <flo@stellbr.ink>"
+    topics = ("conan", "tinyobjloader", "wavefront", "geometry")
+    exports = "LICENSE"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = self.name + '-' + self.version
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        if self.settings.os == "Windows" and self.options.shared:
+            cmake.definitions["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = True
+        cmake.definitions["CMAKE_INSTALL_DOCDIR"] = "licenses"
```

Comment:
does it only install the license or are there more files?

Reply:
Just the license. I agree that this is kind of hacky.
We could also copy the license explicitly and delete the default folder.

Reply:
if it's only the license, I see no problem using this approach. Actually, I prefer using cmake, because for future versions we don't need to check if the upstream changed the file path.

Reply:
Great :)